src/python/pants/backend/shell/shell_command.py

```diff
@@ -231,6 +231,7 @@ async def prepare_shell_command_process(
         output_files=output_files,
         timeout_seconds=timeout,
         working_directory=working_directory,
+        level=LogLevel.DEBUG,
```

Comment:
Wait but this defaults to `INFO`. Are you sure this is the right fix?

Reply:
...oh. Hm. You're right.

So the issue on #16825 is probably related to the orphaned `setup_sandbox` workunit instead.